gatsby-node.esm.js

```diff
@@ -56,18 +52,17 @@ function getRepoName(contestNode) {
 }
 
 async function fetchReadmeMarkdown(contestNode) {
-  const { data } = await octokit.request("GET /repos/{owner}/{repo}/readme", {
-    owner: process.env.GITHUB_CONTEST_REPO_OWNER,
-    repo: `${getRepoName(contestNode)}`,
-    headers: {
-      accept: "application/vnd.github.v3.html+json",
-    },
-  });
-
+  const response = await fetch(
```

Comment:
.. how does this authenticate (does it need to)? The raw request won't work on non-public repos (are those filtered out by this point?)

Reply:
It doesn't authenticate. It is just for pulling in READMEs from contest repos, which are public.

Reply:
Riight— thanks! 😅

Reply:
<img width="845" alt="image" src="https://user-images.githubusercontent.com/96088385/181682270-674017dc-6b4f-4201-b55d-e67a0c72dbe2.png">
Here is how it will look if we ever have a private contest repo for whatever reason. If we want to handle showing private contests on our site eventually, we'll have to rework all of this.